lib/debug/server_dap.rb

```diff
@@ -437,10 +437,12 @@ def process
           expr = req.dig('arguments', 'expression')
           if /\A\s*,(.+)\z/ =~ expr
             dbg_expr = $1
+            commands = dbg_expr.split(';;')
+            commands.each { |cmd| @q_msg << cmd }
+
```

Comment:
String#split accepts a block since [Ruby 2.6](https://bugs.ruby-lang.org/issues/4780).
``dbg_expr.split(';;') { |cmd| @q_msg << cmd }``

Reply:
TIL this usage! Thank you 😄